### PR TITLE
Added metadata to theme_glyphs

### PR DESCRIPTION
**Pull request by tzhang-dev**
Added metadata for examples\plotting\theme_glyphs.py .
Relate to issue https://github.com/bokeh/bokeh/issues/11765

### CODE REVIEW

File: examples/plotting/theme_glyphs.py [hunk 1]
```diff
@@ -1,3 +1,12 @@
+'''This example shows how to to create and use your custom theme in a Bokeh plot with JSON.
```
Comment (mosc9575) [inline examples/plotting/theme_glyphs.py]:
```suggestion
'''This example shows how to create and use your custom theme in a Bokeh plot with JSON.
```

File: examples/plotting/theme_glyphs.py [hunk 2]
```diff
@@ -1,3 +1,12 @@
...
+
+.. bokeh-example-metadata::
+    :apis: bokeh.themes.Theme
+    :refs: :ref:`_ug_styling_using_themes_custom`
+    :keywords: Custom Theme, JSON
```
Comment (mosc9575) [inline examples/plotting/theme_glyphs.py]:
```suggestion
    :keywords: styling, Custom Theme, JSON
```

**Comment by tzhang-dev** [comment]
Hi @mosc9575 , thanks for your review. I accepted the modification. Waiting for more advice.

File: examples/plotting/theme_glyphs.py [hunk 3]
```diff
@@ -1,3 +1,12 @@
+'''This example shows how to to create and use your custom theme in a Bokeh plot with JSON.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.themes.Theme
+    :refs: :ref:`_ug_styling_using_themes_custom`
```
Comment (bryevdv) [inline examples/plotting/theme_glyphs.py]: leading underscore is only for when the anchor is defined, not when it is used
```suggestion
    :refs: :ref:`ug_styling_using_themes_custom`
```

File: examples/plotting/theme_glyphs.py (hunk 2 above)
Comment (bryevdv) [inline examples/plotting/theme_glyphs.py]: we tend to use keywords sparingly, and keep lowercase
```suggestion
    :keywords: theme
```

File: examples/plotting/theme_glyphs.py [hunk 4]
```diff
@@ -1,3 +1,12 @@
...
+    :keywords: Custom Theme, JSON
+
+'''
+
```
Comment (bryevdv) [inline examples/plotting/theme_glyphs.py]: no blank line between docstring an first import
```suggestion
```

**Review by bryevdv — CHANGES_REQUESTED** [review]
Thanks @yah-yuan just a few final comments then this will be good to merge

**Comment by bryevdv** [comment]
Hi @yah-yuan do you expect to be able to continue with this PR?

**Comment by tzhang-dev** [comment]
Hi @bryevdv , sorry that I miss the last notification. And thanks for the document suggestions and I have accepted them.

I noticed some test checks do not pass. Do you think that is a problem?

**Comment by bryevdv** [comment]
I think it is a spurious failure, I have restarted that one test.

**Comment by bryevdv** [comment]
@yah-yuan you marked the conversations resolved, but it does not appear that any of the suggestions were actually applied (there are no new commits)

**Comment by pavithraes** [comment]
@yah-yuan This PR looks close, would you be able to complete it? 

As Bryan said, the previous code review suggest didn't actually get implemented -- you can make these updates in a separate commit if you'd like :)

**Comment by tzhang-dev** [comment]
Hi @pavithraes, sorry for afk for a while. I have committed the suggested changes.

**Review by pavithraes — APPROVED** [review]
Thanks for the updates, @yah-yuan! Looks good to me. :)

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
Thanks @yah-yuan !

**Pull request merged**